hieradata/cluster/rancher/osfamily/RedHat/major/8.yaml

```diff
@@ -0,0 +1,19 @@
+---
+profile::nm::connections:
```

Comment:
I'm not sure if this should be limited to EL8 as the network config should be the same for EL9.

Reply:
where or how in the hierarchy can i put this to grab Alma 8 and 9, but not Centos 7? (i could copy and create a 9.yaml, but the idea is not to repeat the code right?)

Reply:
There is no place in hierachy which would apply this to EL7 as `profile::nm` is only included if the `os.release.major` is NOT `7`.